interfaces/builtin/network_manager.go

```diff
@@ -73,6 +73,9 @@ network packet,
 # from netplan
 /run/NetworkManager/{,**} r,
 
+# Allow writing dhcp lease files outside of the snap
```

Comment:
I don't understand this comment. `/run/NetworkManager/dhcp` looks very much inside the sanp.